pkgs/tools/package-management/nix/README.md

```diff
@@ -0,0 +1,34 @@
+# Steps for Testing New Releases
+
+## Patch Releases or Updating `nixVersions.{latest,git}`
+
+Branch to merge into: **master** or **release-$nixos-version**
+
+Build the affected Nix packages and run their tests on the following platforms: **x86_64-linux**, **aarch64-linux**, **x86_64-darwin**, and **aarch64-darwin**.
+If you lack the necessary hardware for these platforms, you may need to ask others for assistance with the builds.
+Alternatively, you can request access to the Nix community builder for all platforms [here](https://github.com/NixOS/aarch64-build-box) and [here](https://nix-community.org/community-builder/).
+
+To build all dependent packages, use:
+
+```
+nix-review pr <your-pull-request>
+```
+
+And to build all important NixOS tests, run:
+
+```
+# Replace $version with the actual Nix version
+nix-build nixVersions.nix_$version.tests
+```
+
+Be sure to also update the `nix-fallback-paths`:
```

Comment:
I think fallback-paths is supposed to point to outputs of `stable`, or at least that's what I tried to put in code with `addFallbackPathsCheck`, so this should probably move to the next section.

Reply:
If we do a patch release of nixVersions.stable it should be still updated.

Reply:
I clarified this part now.